.github/workflows/builder.yml

```
@@ -11,18 +11,16 @@ env:
   ALPINE_LATEST: "3.22"
   DEBIAN_LATEST: "trixie"
   UBUNTU_LATEST: "24.04"
-  RASPBIAN_LATEST: "trixie"
   PYTHON_LATEST: "3.13"
 
 jobs:
   init:
     name: Initialize build
-    runs-on: &runs-on-ubuntu ubuntu-24.04
+    runs-on: ubuntu-24.04
```

```suggestion
    runs-on: &runs-on-ubuntu ubuntu-24.04
```

If this is reverted, it can be reused in the `matrix.include` block.